Cubemap.cpp

```diff
@@ -675,7 +675,16 @@ void Cubemap::computeMainLightDirection ()
     colorAvg[1]= pixels[maxPixelOffset + 1];
     colorAvg[2]= pixels[maxPixelOffset + 2];
 
-    texelCoordToVect(maxFace, (float)x, (float)y, width, &direction[0], 1);
+    if (width > 1){
+        // divide by Zero with width == 1
+        texelCoordToVect(maxFace, (float)x, (float)y, width, &direction[0], 1);
+    }
+    else{
+        direction[0] = 0.0f;
```

Comment:
Could be be useful. (was the all black or all white env map)
 What's current way of logging warning so that we have trace ? std::cerr ?